content/administration/on_premise/deploy.rst

```diff
@@ -600,6 +600,145 @@ It should be stored securely, and should be generated randomly e.g.
+   internet until this password is set to secure the database.
+
+The master password is specified in the Odoo configuration file (`odoo.conf` or `odoorc` (hidden
+file)). The Odoo master password is needed to modify, create, or delete a database through the
+graphical user interface (GUI).
+
+Locate configuration file
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, :guilabel:`Open` the Odoo configuration file (`odoo.conf` or `odoorc` (hidden file)).
+
+.. tabs::
+
+   .. tab:: Windows
+
+      The configuration file is located at: `c:\\ProgramFiles\\Odoo{VERSION}\\server\\odoo.conf`
+
+   .. tab:: Linux
+
+      Depending on how Odoo is installed on the Linux machine the configuration file is located in
+      different places:
+
+      - Package installation: `/etc/odoo.conf`
+      - Source installation: `~/.odoorc`
+
+Change old password
+~~~~~~~~~~~~~~~~~~~
+
+Then, modify the old password in the configuration file to a temporary password.
+
+.. tabs::
+
+   .. tab:: Graphical user interface
+
+      After locating the configuration file open it using a graphical user interface (GUI). This can
+      be achieved by simply double clicking on the file and the device should have a default
+      :abbr:`GUI (graphical user interface)` to open the file with.
+
+      Next, modify the line to the master password `admin_passwd = $pbkdf2-sha…` to `admin_passwd =
+      newpassword1234`. This password can be anything just as long as it is saved temporarily.
+      Modify all characters after the `=`.
+
+      .. example::
+         The line appears like this:
+         `admin_passwd =
+         $pbkdf2-sh39dji295.59mptrfW.9z6HkA$w9j9AMVmKAP17OosCqDxDv2hjsvzlLpF8Rra8I7p/b573hji540mk/.3ek0lg%kvkol6k983mkf/40fjki79m`
+
+      To reset the master password delete everything after the `=`
+
+      .. example::
+         The line appears like this: `admin_passwd = newpassword1234`
+
+
+   .. tab:: Command-line interface
+
+      Using the following Unix command modify the line to the master password.
+
+      Connect to the Odoo server's terminal via Secure Shell (SSH) protocol and edit the
+      configuration file. To modify the configuration file type: :command:`sudo nano /etc/odoo.conf`
+
+      After opening the configuration file, modify the line to the master password `admin_passwd =
+      $pbkdf2-sha…` to `admin_passwd = newpassword1234`. This password can be anything just as long
+      as it is saved temporarily. Modify all characters after the `=`.
+
+      .. example::
+         The line appears like this:
+         `admin_passwd =
+         $pbkdf2-sh39dji295.59mptrfW.9z6HkA$w9j9AMVmKAP17OosCqDxDv2hjsvzlLpF8Rra8I7p/b573hji540mk/.3ek0lg%kvkol6k983mkf/40fjki79m`
+
+      To reset the master password delete everything after the `=`
+
+      .. example::
+         The line appears like this: `admin_passwd = newpassword1234`
+
+.. important::
+   It is essential that the password is changed to something else rather than triggering a new
+   password reset.
```

Comment:
I'm having trouble understanding what this sentence means - consider rewording a bit to make it clear (unless I'm totally missing the point)